corehq/messaging/scheduling/tests/test_content.py

```diff
@@ -221,3 +246,11 @@ class Content(AbstractContent):
 @unregistered_django_model
 class Schedule(AbstractSchedule):
     pass
+
+
+class RecipientWithEmail:
```

Comment:
I believe this could be called, `MockRecipient`?

Reply:
58c7d914700